src/operator/mergeScan.ts

```diff
@@ -9,28 +9,45 @@ import { OuterSubscriber } from '../OuterSubscriber';
 import { InnerSubscriber } from '../InnerSubscriber';
 
 /**
- * @param project
- * @param seed
- * @param concurrent
- * @return {Observable<R>|WebSocketSubject<T>|Observable<T>}
+ * Applies an accumulator function over the source Observable where the
+ * accumulator function itself returns an Observable, then each intermediate
+ * Observable returned is merged into the output Observable.
+ *
+ * <span class="informal">It's like {@link scan}, but the Observables returned
+ * by the accumulator are merged into the outer Observable.</span>
+ *
+ * @example <caption>Count the number of click events</caption>
+ * var clicks = Rx.Observable.fromEvent(document, 'click');
```

Comment:
FYI, I couldn't find any operator examples that are using TypeScript. The only TypeScript I found in the docs were interfaces for some classes, e.g., [Action](http://reactivex.io/rxjs/class/es6/scheduler/Action.js~Action.html). Is this just a change going forward? I'm happy to make that change but just want to confirm.